Include.psm1

```diff
@@ -690,18 +688,9 @@ function Get-ChildItemContent {
 }
 
 filter ConvertTo-Hash { 
-    [CmdletBinding()]
-    $Hash = $_
-    switch ([Math]::Truncate([Math]::Log([Math]::Abs($Hash), [Math]::Pow(1000, 1)))) { 
-        $null { "0  H" }
-        "-Infinity" { "0  H" }
-        0 { "{0:n2}  H" -f ($Hash / [Math]::Pow(1000, 0)) }
-        1 { "{0:n2} KH" -f ($Hash / [Math]::Pow(1000, 1)) }
-        2 { "{0:n2} MH" -f ($Hash / [Math]::Pow(1000, 2)) }
-        3 { "{0:n2} GH" -f ($Hash / [Math]::Pow(1000, 3)) }
-        4 { "{0:n2} TH" -f ($Hash / [Math]::Pow(1000, 4)) }
-        Default { "{0:N2} PH" -f ($Hash / [Math]::Pow(1000, 5)) }
-    }
+    $Units = " kMGTPEZY" # k(ilo) in small letters, see https://en.wikipedia.org/wiki/Metric_prefix
+    $Base1000 = [math]::Abs([math]::truncate([math]::log($_, [Math]::Pow(1000, 1))))
```

Comment:
The removal of the switch was decided against previously but won’t oppose the change. As the negative bug still pertains, this will be rewritten at 12 noon GMT.